Makefile

```diff
@@ -49,3 +51,26 @@ tester-image: ## Build docker image for the tester component
 kind-load: image tester-image ## Load the required image to KinD cluster
 	kind load docker-image $(REPO):latest
 	kind load docker-image $(TESTER_REPO):v1
+
+.PHONY: images
+images: ## Build multi arch docker images for the project
+	docker buildx build \
+		--platform $(SUPPORTED_ARCHS) \
+		--output "type=image,push=false" \
+		--tag $(REPO):$(VERSION) \
+		--tag $(REPO):latest \
+		.
+
+.PHONY: push
+push: images ## Push multi arch docker images to the registry
```

Comment:
This will cause CI to run `docker buildx build` twice, using different `--output` configuration. I think on CI, we should be able to just use `make push`.

Reply:
Yes, but the second run will be using the cache Overall the whole release workflow finishes in 1 minute.